syntax = "proto3";

service MyService {
    rpc TestMessage (Request3) returns (Response3);
      rpc PushCode (Request4) returns (Response4);

}

message Request4 {
  string payload = 1;
}

message Response4 {
  string response = 1;
}

message Request3 {
  string payload = 1;
}

message Response3 {
  string response = 1;
}